src/plone/restapi/tests/test_batching.py

```diff
@@ -172,10 +174,12 @@ def test_contains_correct_batch_of_items(self):
         response = self.api_session.get("/collection?b_start=2&b_size=2")
 
         # Response should contain second batch of items
-        self.assertEqual(
```

Comment:
Thanks for taking care of this!
IMO the test was more descriptive earlier.
I would just have replaced it with:
```
self.assertListEqual(
    [u"/plone/folder/doc-2", u"/plone/folder/doc-3"],
    sorted(result_paths(response.json())),
)
```
or with:
self.assertSetEqual(
    {u"/plone/folder/doc-2", u"/plone/folder/doc-3"},
    set(result_paths(response.json())),
)
```

I am not sure if it make sense to modify the `result_paths` helper to do this already.

Reply:
@ale-rt you's right. This line would not need to be changed. Thanks!

Reply:
Sorry @ale-rt . I thought you were talking about another line. This one needs to be changed. We cannot test with assertEqual because the results are not guaranteed to be the same.

We can have `doc-3` and `doc-4` or `doc-2` and `doc-3` or ...

Reply:
Ah! ok :) Go ahead with your idea, sorry for the noise!